src/js/controller.js

```diff
@@ -0,0 +1,135 @@
+import { Store } from './model';
+import { View } from './view';
+import { Address } from './single-address';
+import { appendChild, toggleEmptyListMessage, setInitialFormValues } from './helpers';
+
+export class Controller {
+
+  constructor() {
+
+    this.HIDDEN_STATE_STRING = 'hidden';
+    
+    this.initialFormState = setInitialFormValues();
+
+    this.store = new Store();
+    this.addressList = this.store.fetchFromLocalStorage();
+    
+    this.view = new View();
+    this.form = this.view.getFormHook();
```

Comment:
Co to robi? Nie widzę żebyś to gdzieś wykorzystywał w kontrolerze.

Reply:
Zapomniałem usunąć ;)